core/ts/__tests__/benchmarks/index.ts

```diff
@@ -0,0 +1,111 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Let's add benchmarks folder to exception for import/no-extraneous-dependencies linter rule in root config instead. And you can remove this line after it.